File: rxbindingdemo/src/main/java/org/videolan/rxbindingdemo/MainActivity.java
package org.videolan.rxbindingdemo;


import android.os.Build;
import android.os.Bundle;
import android.widget.EditText;
import android.widget.TextView;

import com.jakewharton.rxbinding3.view.RxView;

import java.lang.reflect.Method;

import androidx.appcompat.app.AppCompatActivity;
import io.reactivex.functions.Consumer;
import kotlin.Unit;

public class MainActivity extends AppCompatActivity {


    private TextView tv_;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        tv_=findViewById(R.id.tv_);

        RxView.clicks(tv_).subscribe(new Consumer<Unit>() {
            @Override
            public void accept(Unit unit) throws Exception {

            }
        });
    }




}

File: aop/src/main/java/org/videolan/aop/MainActivity.kt
package org.videolan.aop

import android.os.Bundle
import android.view.View
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun login(view: View) {
        LoginManager.isLogin = true;
    }

    @LoginState
    fun jump(view: View) {
        Toast.makeText(this, "2323232", Toast.LENGTH_SHORT).show()
    }
}
